packages/cli/src/lib/quire/index.js

```diff
@@ -185,19 +187,22 @@ async function install(version, options={}) {
 }
 
 /**
- * Install a specific version of `quire-11ty` directly into a quire project
+ * Install `quire-11ty` directly into a quire project
+ *
+ * @TODO refactor this to use the install method with pre and post-install hooks
+ * or steps to prepare the working directory and cleanup on error and completion
  *
  * @param  {String}  projectPath  Absolute system path to the project root
- * @param  {String}  version  Quire-11ty semantic version
  * @param  {Object}  options  options passed from `quire new` command
  * @return  {Promise}
  */
-async function installInProject(projectPath, version, options={}) {
+async function installInProject(projectPath, options = {}) {
```

Comment:
the call to `setVersion` in `installInProject` is setting the `.quire` version to still use latest, rather than passing the specified version from `--quire` flag.

Output when I generate a new pub:
```sh
[CLI:quire] init-starter 
  project root: "blerg" 
  starter: "https://github.com/thegetty/quire-starter-default"
blerg set to use quire-11ty@1.0.0-rc.3  # THIS IS STILL 1.0.0-rc.3
[CLI:quire] installing quire-11ty@1.0.0-rc.5 into blerg
[CLI:quire] installing dev dependencies into quire project
```